### DISCUSSION TITLE
SEO optimizable routes with dynamic routing

### DISCUSSION THREAD

**Discussion by cryptoWizarder** [vercel/next.js#67073] [Help] [answered]
### Summary

Feature request
Is your feature request related to a problem? Please describe.
What I'm trying to accomplish is with dynamic routing have a substructure as so:

Full path - for each subroute we would need a seperate page (specific page component as well)
/[lang]/[overview-slug]/[brand]/[model]/[id]

Note that overview-slug is language specific
/nl/overzicht
/nl/overzicht/audi
/nl/overzicht/audi/a3
/nl/overzicht/audi/a3/12430213-123-123123-123

Describe the solution you'd like
previous question

Describe alternatives you've considered
Used to work with next-routes, but currently unable to export

Additional context
Basic repository for reproduction:
https://github.com/lizy-lab/lizy-basic-project

**Comment by 0xrustySol** [comment, marked as answer]
how would one go about conflict routes in this scenario?

Or would you do this on page level to check the route and show a different page-component?

**Reply by cryptoWizarder** [reply]
I am not sure about that could you let me know in more detail?
Thanks.

**Reply by 0xrustySol** [reply]
Let me check again let me know your  contact info.
